cobigen-templates/templates-devon4j/src/main/templates/crud_ngrx_client_app/templates/src/app/home/${variables.etoName#lower_case}/components/${variables.etoName#lower_case}-grid/${variables.etoName#lower_case}-grid.component.ts.ftl

```diff
@@ -61,7 +58,7 @@ export class ${variables.etoName?cap_first}GridComponent implements OnInit {
     ];
 
   totalItems: number;
-  pageSize: number = 8;
+  pageSize = 8;
```

Comment:
Here you are also removing the type (`number`)? Is it because of the merger?

Reply:
See answer above :-)